src/main/java/converter/UserToStringConverter.java

```diff
@@ -19,6 +18,7 @@ public String convert(User user) {
         joiner.add(user.getPassword());
         joiner.add(user.getName());
         joiner.add(user.getSurname());
+        joiner.add(user.getRole().toString());
```

Comment:
maybe it will work without toString() call

Reply:
without it dont work, may be I shoud find another way